src/components/Calculator.js

```diff
@@ -1,66 +1,58 @@
         super(props);
-        this.display = this.display.bind(this);
-        this.clearScreen = this.clearScreen.bind(this);
-        this.calculate = this.calculate.bind(this);
-        this.toggleSign = this.toggleSign.bind(this);
+        this.state = {
+            total: '',
+            next: '',
+            operation: '',
+        };
+        this.handleClick = this.handleClick.bind(this);
     }
-    display() {
-
-    }
-    clearScreen() {
-
-    }
-    calculate() {
-
+    handleClick = (e) => {
+        console.log({...this.state})
+        this.setState(calculate({...this.state}, e.target.value));
     }
-    toggleSign() {
-
-    }
-
     render() {
         return (
             <table className='calculator'>
                 <tbody>
                     <tr>
-                        <td colSpan={4}><input className='displayBox' type='text' id='result' value={0} disabled/></td>
+                        <td colSpan={4}><input className='displayBox' type='text' value={this.state.next || this.state.total || '0'} disabled/></td>
                     </tr>
                     <tr>
-                    <td><input className='button' type='button' value='AC' onClick= {this.clearScreen()}/></td>
-                    <td><input className='button' type='button' value='+/-' onClick= {this.toggleSign()}/> </td>
-                    <td><input className='button' type='button' value='%' onClick= {this.display('%')}/></td> 
-                    <td><input className='button orange' type='button' value='/' onClick= {this.display('/')}/></td> 
+                    <td><input className='button' type='button' value='AC' onClick= {this.handleClick}/></td>
+                    <td><input className='button' type='button' value='+/-' onClick= {this.handleClick}/> </td>
+                    <td><input className='button' type='button' value='%' onClick= {this.handleClick}/></td> 
+                    <td><input className='button orange' type='button' value='÷' onClick= {this.handleClick}/></td> 
                     </tr>
                     <tr>
-                    <td><input className='button' type='button' value='7' onClick= {this.display('7')}/></td>
-                    <td><input className='button' type='button' value='8' onClick= {this.display('8')}/></td> 
-                    <td><input className='button' type='button' value='9' onClick= {this.display('9')}/></td> 
-                    <td><input className='button orange' type='button' value='X' onClick= {this.display('X')}/></td> 
+                    <td><input className='button' type='button' value={7} onClick= {this.handleClick}/></td>
+                    <td><input className='button' type='button' value={8} onClick= {this.handleClick}/></td> 
+                    <td><input className='button' type='button' value={9} onClick= {this.handleClick}/></td> 
+                    <td><input className='button orange' type='button' value='x' onClick= {this.handleClick}/></td> 
```

Comment:
- [ ] Kindly fix the error in the image below. The error only comes when I click the multiplication sign.

![math](https://user-images.githubusercontent.com/53124548/170507636-b62b80d9-dd50-4cb9-9ea4-9c2979168046.png)

Reply:
@elisha2kyakpo1  Hello, I don't see that issue on my browser. So don't know how to fix. I used 'x' as a value for that button, but you see 'X' in the console. I get confused. 😕